corehq/apps/locations/tests/test_reupholster.py

```diff
@@ -29,28 +28,28 @@ def test_path(self):
 
     def test_lineage(self):
         for i in range(len(self.all_locs)):
-            expected_lineage = list(reversed(self.all_loc_ids[:i+1]))
-            self.assertEqual(expected_lineage, get_lineage_from_location_id(self.all_loc_ids[i]))
-            self.assertEqual(expected_lineage, get_lineage_from_location(self.all_locs[i]))
+            # a location should not be included in its own lineage
+            expected_lineage = list(reversed(self.all_loc_ids[:i]))
+            self.assertEqual(expected_lineage, self.all_locs[i].lineage)
 
     def test_move(self):
-        original_parent = self.all_locs[1]
-        new_state = make_loc('New York', type='state')
-        new_district = make_loc('NYC', type='block', parent=original_parent)
-        self.assertEqual(original_parent._id, new_district.sql_location.parent.location_id)
-        # this is ugly, but how it is done in the UI
-        new_district.lineage = get_lineage_from_location(new_state)
-        new_district.save()
-        self.assertEqual(new_state._id, new_district.sql_location.parent.location_id)
+        original_parent = self.all_locs[1].sql_location
+        new_parent = make_loc('New York', type='state').sql_location
+        district = make_loc('NYC', type='block', parent=original_parent.couch_location).sql_location
+        self.assertEqual(original_parent.site_code, district.couch_location.parent.site_code)
+
+        district.parent = new_parent
+        district.save()
+        self.assertEqual(new_parent.site_code, district.couch_location.parent.site_code)
```

Comment:
Yeah, I suppose it couldn't hurt.  These changes are a little scary, so I'm happy to test whatever makes sense.